README.md

```diff
@@ -1,9 +1,27 @@
+Simple Blog
+=
+
+Technologies used:
+-
+
+* Ruby 2.6.4
+
+* PostgreSQL 
+
+* Docker-compose
+
+* The gems used can be found in Gemfile
+
 Start application:
-===
-* `docker-compose build`
+-
+
+* Download the application to your device or use the: `git clone https://github.com/makarenkoj/simple_blog.git`
+ in your terminal(command line)
+
+* Further in the terminal in the application folder run the following commands: `docker-compose build`
 
-* `docker-compose up`
+* To get started: `docker-compose up`
 
-* `sudo service postgresql stop`
+* If you get an error on your device stating that postgresql took up the port, use: `sudo service postgresql stop`
```

Comment:
postgres might be different in other os. Also port might be taken by something else than pg. Id remove this line as this is also obvious that the port has to be released